crates/tests/databases-tests/src/cockroach/query_tests.rs

```diff
@@ -45,6 +45,12 @@ mod basic {
         let result = run_query(create_router().await, "native_queries/embedded_variable").await;
         insta::assert_json_snapshot!(result);
     }
+
+    #[tokio::test]
+    async fn select_composite_variable() {
+        let result = run_query(create_router().await, "select_composite_variable").await;
```

Comment:
TODO: Leave out this test, since cockroach does not support nested user defined types.